pkg/pods/pod.go

```diff
@@ -529,6 +529,12 @@ func (pod *Pod) setupConfig(manifest Manifest, launchables []launch.Launchable)
 	}
 
 	for _, launchable := range launchables {
+		// we need to remove any unset env vars
+		err = os.RemoveAll(launchable.EnvDir())
+		if err != nil && !os.IsNotExist(err) {
```

Comment:
Since `RemoveAll` [returns no error if the path doesn't exist](https://golang.org/pkg/os/#RemoveAll), I suspect the `!os.IsNotExist` is unnecessary, but I see no harm in leaving it in (also don't have time to check what happens if a parent doesn't exist, so maybe it's needed)